pkg/logger/logger.go

```diff
@@ -0,0 +1,103 @@
+package logger
```

Comment:
unit tests for this?

Why not using https://github.com/uber-go/zap ?

Reply:
I opted for zerolog for this project because it is faster, and I am more familiar with its simplicity. While zap is more powerful and customizable, and I acknowledge its capabilities, for the scale of this small project, the speed and ease of use offered by zerolog were more fitting. In larger projects, I would definitely consider using zap for its extensive features and customization options.

Reply:
Added tests